docs/5.api/rpc/introduction.md

```diff
@@ -11,11 +11,16 @@ import ContactUs from '@site/components/ContactUs.mdx';
 The RPC API allows you to communicate directly with the NEAR network. For example,
 tools such as [near-api-js](/tools/near-api-js/quick-reference) are just abstractions making RPC calls.
 
+## Free Public RPC
```

Comment:
This seams only relevant to lava and not the general concept of RPC (e.g. it does not apply for https://rpc.mainnet.near.org/). Therefore, it does not belong in the general documentation.